src/styles/Book.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --bubble-gum-pink: #ff5ab7;
+  --black: #1e1e1e;
+  --pinkish-grey: #c7c7c7;
+  --very-light-pink: #c4c4c4;
+  --pale-grey: #f5f6fa;
+  --white: #fafafa;
+  --white-two: #fff;
+  --white-three: #e8e8e8;
+  --black-5: rgba(0, 0, 0, 0.05);
+  --medium-blue: #307bbe;
+  --azure: #0290ff;
+  --cool-blue: #4386bf;
+}
+
+button {
+  background-color: transparent;
+  border: none;
+}
+
+.line-btn {
+  width: 0.1rem;
+  height: 2rem;
+  border: solid 1px var(--very-light-grey);
+}
+
+.border {
+  border: 1px solid #e8e8e8;
+  box-shadow: 0 0 2rem 1rem rgba(204, 204, 204, 0.05);
+  border-radius: 0.5rem;
+}
+
+.bc-azure {
+  background-color: var(--azure);
+}
+
+.c-white-two {
+  color: var(--white-two);
+}
+
+.bc-white-two {
+  background-color: var(--white-two);
+}
```

Comment:
Kindly ensure that your design is as close as possible with the design given. you did well, please i'd suggest you add more padding to the boxes and also ensure that they are on the same line vertically as i showed in the arrows below 😄 

Your design :
![image](https://github.com/Abdelaziz-Ali/bookstore/assets/20537712/939fc810-545d-4c71-a01d-6d8460f91011)

Design given :
![image](https://github.com/Abdelaziz-Ali/bookstore/assets/20537712/dfb01b28-cf72-4094-a05b-a5d2ac6c13b4)